pandas/core/series.py

```diff
@@ -4630,6 +4630,34 @@ def isin(self, values) -> "Series":
         4     True
         5    False
         Name: animal, dtype: bool
+        
+        New behaviour from pandas v1.2.0 caused some tests to fail because users where unaware of this change.
+        
+        Before v1.2.0:
+        
+        >>>import pandas as pd
+        pd.Series([0]).isin(['0']) 
+        # 0    True
+        # dtype: bool
+        pd.Series([1]).isin(['1']) 
+        # 0    True
+        # dtype: bool
+        pd.Series([1.1]).isin(['1.1']) 
+        # 0    True
+        # dtype: bool
+        
+        From v1.2.0
+        
+        >>>import pandas as pd
+        pd.Series([0]).isin(['0']) 
+        # 0    False
```

Comment:
What are the hashes for?

Reply:
It was meant to be a comment, but you are right, it's better if I remove them.